spec/acceptance/docker_spec.rb

```diff
@@ -9,12 +9,12 @@
   raise 'Could not retrieve ip address for Windows box' if result.exit_code != 0
   ip = result.stdout.split("\n")[0].split(':')[1].strip
   @windows_ip = ip
-  docker_arg = "docker_ee => true, extra_parameters => '\"insecure-registries\": [ \"#{@windows_ip}:5000\" ]'"
+  docker_args = "docker_ee => true, extra_parameters => '\"insecure-registries\": [ \"#{@windows_ip}:5000\" ]'"
+  root_dir = 'C:/Users/Administrator/AppData/Local/Temp'
```

Comment:
We could probably just make this more generic, seeing it's going to be used on both Windows and Unix:
```ruby
root_dir = '/foo/bar'
```
We may need to unset this in an `:after` block too, depending on the impact. However, we're currently setting `root-dir` with Windows path on Unix systems, so we may be getting away with it if `docker_args` is being set in every test (_something to look in to_)?

Reply:
Hi @sanfrancrisko 
The `root_dir` is set with `'C:/Users/Administrator/AppData/Local/Temp'` on windows loses and with `'root'`. But I think we can make the new test to be more generic such as:
`class { 'docker': #{docker_args}, root_dir => \"#{root_dir}/foo/bar\"}`

Kind regards,
Adrian IURCA